### PR TITLE
Doc queue #56: expand description of a few VFS modules

### PR DESCRIPTION
**Pull request by Mrt134**
HTML build test: no issues.

### CODE REVIEW

**Comment by Mrt134** [comment]
Cherry-picked last commit from master branch PR and unified tables between both versions.

File: userguide/sharing.rst [hunk 1]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
-   |                     |                                                                                                                                            |
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | Value               | Description                                                                                                                     |
+   |                     |                                                                                                                                 |
+   +=====================+=================================================================================================================================+
+   | acl_tdb             | Store NTFS ACLs in a tdb file to enable full mapping of Windows ACLs.                                                           |
```
Comment (wblock) [inline userguide/sharing.rst]: Use :file: markup on "tdb".

File: userguide/sharing.rst [hunk 2]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   | cap                 | Translate filenames to and from the CAP encoding format, commonly used in Japanese language environments.                       |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | catia               | Improve Mac interoperability by translating characters that are unsupported by Windows.                                         |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | commit              | Track the amount of data written to a file and synchronizes it to disk when a specified amount accumulates.                     |
```
Comment (wblock) [inline userguide/sharing.rst]: s/synchronizes/synchronize/

File: userguide/sharing.rst [hunk 3]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   | dirsort             | Sort directory entries alphabetically before sending them to the client.                                                        |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | expand_msdfs        | Enable support for Microsoft Distributed File System (DFS).                                                                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | extd_audit          | Send :guilabel:`audit` logs to both syslog and the Samba log files.                                                             |
```
Comment (wblock) [inline userguide/sharing.rst]: I don't think guilabel is correct here.  I think it does not need any markup at all, these are just audit logs.

File: userguide/sharing.rst [hunk 4]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | fake_perms          | Allow roaming profile files and directories to be set as read-only.                                                             |
```
Comment (wblock) [inline userguide/sharing.rst]: s/as/to/

File: userguide/sharing.rst [hunk 5]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | fruit               | Enhance macOS support by providing the SMB2 AAPL extension and Netatalk interoperability.                                       |
+   |                     | Automatically loads *catia* and *streams_xattr* but read the caveat in NOTE below table.                                        |
```
Comment (wblock) [inline userguide/sharing.rst]: s/ but/, but/
", but see the warning below." (make "warning" a link to the note)

File: userguide/sharing.rst [hunk 6]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   | ixnas               | Experimental module to improve ACL compatibility with Windows and store DOS attributes as file flags.                           |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | linux_xfs_sgid      | Used to work around an old Linux XFS bug.                                                                                       |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | media_harmony       | Allow Avid editorial workstations to share a network drive.                                                                     |
```
Comment (wblock) [inline userguide/sharing.rst]: s/editorial/editing/

File: userguide/sharing.rst [hunk 7]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | streams_depot       | **Experimental** module to store alternate data streams in a central directory.                                                 |
+   |                     | The association with the primary file can be lost due to inode numbers changing when a directory is copied to a new location    |
+   |                     | See `<https://marc.info/?l=samba&m=132542069802160&w=2>`__.                                                                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | streams_xattr       | Enabled by default. Enable storing NTFS alternate data streams in the file system.                                              |
```
Comment (wblock) [inline userguide/sharing.rst]: Swap sentences (tell what it does first, then whether it is enabled).

File: userguide/sharing.rst [hunk 8]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | syncops             | Ensure metadata operations are performed synchronously.                                                                         |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | time_audit          | Log system calls that take longer than the number of defined milliseconds.                                                      |
```
Comment (wblock) [inline userguide/sharing.rst]: "defined number of"

File: userguide/sharing.rst [hunk 9]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | unityed_media       | Allow multiple Avid clients to share a network drive.                                                                           |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | virusfilter         | This extremely **experimental** module is still under development and does not work at this time.                               |
```
Comment (wblock) [inline userguide/sharing.rst]: Why are we listing this if people can't use it?

File: userguide/sharing.rst [hunk 10]
```diff
@@ -1006,131 +1006,136 @@ for more details.
...
+   | winmsa              | Emulate the Microsoft *MoveSecurityAttributes=0* registry option. Moving files or directories sets the ACL for file and         |
+   |                     | directory hierarchies to inherit from the destination directory.                                                                |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | worm                | Control the writability of files and folders depending on their change time and an adjustable grace period.                     |
+   |                     |                                                                                                                                 |
+   +---------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | xattr_tdb           | Store Extended Attributes (EAs) in a tdb file so they can be used on filesystems which do not provide support for EAs.          |
```
Comment (wblock) [inline userguide/sharing.rst]: :file: markup on "tdb".

File: userguide/sharing.rst (hunk 9 above)
Comment (Mrt134) [reply userguide/sharing.rst]: Looks like it is going to be removed for U4 - will update PR when that change is confirmed.

File: userguide/sharing.rst [hunk 11]
```diff
@@ -1006,135 +1006,142 @@ for more details.
...
+   | winmsa               | Emulate the Microsoft *MoveSecurityAttributes=0* registry option. Moving files or directories sets the ACL for file and         |
+   |                      | directory hierarchies to inherit from the destination directory.                                                                |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | worm                 | Control the writability of files and folders depending on their change time and an adjustable grace period.                     |
+   |                      |                                                                                                                                 |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | xattr\_\ :file:`tdb` | Store Extended Attributes (EAs) in a tdb file so they can be used on filesystems which do not provide support for EAs.          |
```
Comment (wblock) [inline userguide/sharing.rst]: The first tdb is part of a module name and should not have markup.  The second tdb should have :file: markup, as it did in the other version of this patch.

File: userguide/sharing.rst [hunk 12]
```diff
@@ -1006,135 +1006,142 @@ for more details.
...
-   +---------------------+--------------------------------------------------------------------------------------------------------------------------------------------+
-
-
-.. note:: Be careful when using multiple SMB shares, some with and some
-   without *fruit*. macOS clients negotiate SMB2 AAPL protocol
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | Value                | Description                                                                                                                     |
+   |                      |                                                                                                                                 |
+   +======================+=================================================================================================================================+
+   | acl\_\ :file:`tdb`   | Store NTFS ACLs in a tdb file to enable full mapping of Windows ACLs.                                                           |
```
Comment (wblock) [inline userguide/sharing.rst]: No :file: markup in the name, just for the mention as a file extension in the description.

**Comment by Mrt134** [comment]
This is ready to merge: PRs for https://jira.ixsystems.com/browse/NAS-100532 will track further VFS object changes for 11.3.

**Pull request merged**